### ISSUE TITLE
WARNING:tensorflow:Gradients do not exist for variables

### ISSUE THREAD

**Issue by elpapi42** [ElPapi42/DeepBay#8]
When Training a DenseBlock instance, it output this warning:

```
WARNING:tensorflow:Gradients do not exist for variables ['dense_block_1/batch_normalization_1/gamma:0', 'dense_block_1/batch_normalization_1/beta:0'] when minimizing 
the loss.
```

It is caused by the Bath Normalization layer, check what is going on.
